Kernel/Process.cpp

```diff
@@ -4772,6 +4772,14 @@ int Process::sys$unveil(const Syscall::SC_unveil_params* user_params)
     if (path.value().is_empty() || path.value().characters()[0] != '/')
         return -EINVAL;
 
+    auto custody_or_error = VFS::the().resolve_path_without_veil(path.value(), root_directory());
+    if (custody_or_error.is_error())
+        // FIXME Should this be EINVAL?
+        return custody_or_error.error();
```

Comment:
Please use `{` and `}` whenever the contents of a block spans multiple lines, even if some of them are comments.